wagtail/sites/views.py

```diff
@@ -1,5 +1,6 @@
-from django.utils.translation import gettext_lazy
+from django.utils.translation import gettext_lazy as _
```

Comment:
I think it makes sense to call ``gettext_lazy`` with something different than ``gettext`` to make it less likely to use the wrong one. What do you think?

Reply:
The makemessages command only recognises `_` and the original function names, so we don't have a lot of freedom here. (I seem to remember us trying to use `_` and `__` in the distant past, and it not working at all...) From what I can see, we've adopted an informal convention of using `_` for whichever one is most useful within that file:

* `from django.utils.translation import gettext_lazy as _` has 94 matches across the codebase
* `from django.utils.translation import gettext as _` has 53 matches

and if the generic views are doing their job properly, there'll be a lot more strings in class-level configuration than inside methods. (Besides, is there any real penalty for using `gettext_lazy` when `gettext` would have worked...?)

Reply:
I could not find anything against using `gettext_lazy`. Do be doubly sure I tester the changes with a different admin language and it is all working as advertised/expected.